src/ImmersedBoundaries/partial_cell_bottom.jl

```diff
@@ -1,9 +1,10 @@
 using Oceananigans.Utils: prettysummary
 using Oceananigans.Fields: fill_halo_regions!
-using Oceananigans.Grids: bottommost_active_node
+using Oceananigans.Grids: bottommost_active_node, AbstractStaticGrid
 using Printf
 
-import Oceananigans.Operators: Δrᶜᶜᶜ, Δrᶜᶜᶠ, Δrᶜᶠᶜ, Δrᶜᶠᶠ, Δrᶠᶜᶜ, Δrᶠᶜᶠ, Δrᶠᶠᶜ, Δrᶠᶠᶠ
+import Oceananigans.Operators: Δrᶜᶜᶜ, Δrᶜᶜᶠ, Δrᶜᶠᶜ, Δrᶜᶠᶠ, Δrᶠᶜᶜ, Δrᶠᶜᶠ, Δrᶠᶠᶜ, Δrᶠᶠᶠ,
```

Comment:
It seems like a lot of work is being done to introduce the variable r, an intermediate variable for depth, and then you assign z to r.  Is this more than cosmetic?

Also, which line exactly fixes the zspacing()? I see zspacings appearing in th tests but not the function intself.